YoutubeExplode/Bridge/PlaylistExtractor.cs

```diff
@@ -20,6 +20,20 @@ internal partial class PlaylistExtractor
             .GetPropertyOrNull("items")
     );
 
+    private JsonElement? TryGetPlaylistProperty() => Memo.Cache(this, () =>
+        _content
+            .GetPropertyOrNull("contents")?
+            .GetPropertyOrNull("twoColumnWatchNextResults")?
+            .GetPropertyOrNull("playlist")?
+            .GetPropertyOrNull("playlist") ??
+
+        _content
+            .GetPropertyOrNull("continuationContents")?
+            .GetPropertyOrNull("playlistPanelContinuation")
+
+
+    );
```

Comment:
```suggestion
            .GetPropertyOrNull("playlistPanelContinuation")
    );
```

Remove extra linebreak